styles.js

```diff
@@ -8,7 +8,15 @@ export default createGlobalStyle`
   }
 
   body {
-    margin: 0;
+    margin: 6%;
     font-family: system-ui;
   }
 `;
+
+// body {
+//   margin: 0;
+//   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica,
+//     Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+// }
+// // more global styles here...
+// `;
```

Comment:
Avoid unnecessary overloads delete commented out code. 

```suggestion
```

Reply:
Thanks. I deleted the outcommented code.